c2corg_ui/templates/helpers/edit.html

```diff
@@ -0,0 +1,33 @@
+<%def name="show_editing_associated_waypoints(model, type='waypoints')">\
+  <section ng-show="${model}.associations.${type}.length > 0" class="associated-section">
+    <label translate>Associated waypoints</label>
+    <article class="associated-documents">
+      <div class="list-item"
+           ng-repeat="wp in ${model}.associations.${type} | orderBy: 'waypoint_type'"
+           ng-class="{'main-waypoint': ${model}.main_waypoint_id === wp.document_id}">
+        <app-card app-card-doc="wp" app-card-disable-click="true"></app-card>
+        % if model == 'route':
+          <div ng-if="${model}.main_waypoint_id === wp.document_id" ng-init="${model}.main_waypoint_title = wp.locales[0].title" translate>main waypoint</div>
+          <button class="btn btn-sm btn-default" translate
+                  ng-click="${model}.main_waypoint_id = wp.document_id; ${model}.main_waypoint_title = wp.locales[0].title"
+                  ng-if="${model}.main_waypoint_id != wp.document_id">
+            set as main waypoint
```

Comment:
For constants, yes. But I don't see any sense in "translating" (copy&pasting) the English language strings again.